text/0000-long-error-codes-explanation-normalization.md

```diff
@@ -0,0 +1,116 @@
+
+Start Date: 2016-01-04
+
+RFC PR:
+
+Rust Issue: N/A
+
+# Summary
+
+Long error codes explanations haven't been normalized yet. This RFC intends to do it in order to uniformize them.
```

Comment:
The words 'normalized' and 'uniformize' are jargony. It's not clear to me what it means for an error to be normalized, though from the context of wanting to 'uniformize' them I understand that the problem is that Rust error explanations aren't consistent. Can you restate this with more plain language? What is inconsistent about them?